Lib/dataclasses.py

```diff
@@ -1527,10 +1527,11 @@ class C(Base):
     seen = set()
     annotations = {}
     defaults = {}
+    any_marker = object()
```

Comment:
I'll reserve a full review until I've had time to read PEP 749, but couldn't the `object()` used here be a module-level "constant", instead of being allocated every time?